src/ApiNavigation.js

```diff
@@ -841,13 +841,13 @@ export class ApiNavigation extends AmfHelperMixin(LitElement) {
       return;
     }
     const ekey = this._getAmfKey(this.ns.aml.vocabularies.apiContract.endpoint);
-    const endpoint = this._ensureArray(data[ekey]);
+    let endpoint = this._ensureArray(data[ekey]);
+    if (this.rearrangeEndpoints) {
+      endpoint = this._rearrangeEndpoints(endpoint);
```

Comment:
When the `rearrangeEndpoints` property is set and the `endpoint` is undefined then `_rearrangeEndpoints()` function returns empty array. Then there's a condition that tests whether `endpoint` is set which in this case would exist because empty array was created. I prefer this function to return `undefined` or `null` to never execute this code block.

Reply:
Good point